src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -486,6 +487,29 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
             Uri.TryCreate(finalUrl, UriKind.Absolute, out Uri result);
             return result;
         }
+
+        /// <summary>
+        /// Retains characters within ascii range 1-127 and returns
+        /// an escaped representation of the string
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns></returns>
+        internal static string EscapeForUrl(string str)
```

Comment:
>EscapeForUrl [](start = 31, length = 12)

There may also be some value in having a Dictionary of substitutions (non-breaking space to space, middle dot to a star, etc.) to try to preserve more natural formatting

Reply:
This seems like something we would want at least for the characters we use internally, like the ● bullet

Reply:
I think it's an interesting idea, although these strings do get truncated. For the bullet case, it's the difference between `patterns: Modify a b c...see more in the a11y test file` and `patterns: {substitution} Modify a b c...see more in the a11y test file`. I wonder whether it's more complexity than we need